.azure-pipelines/all_pr.yml

```diff
@@ -14,10 +14,23 @@ trigger: none
 variables:
   PIP_CACHE_DIR: $(Pipeline.Workspace)/.cache/pip
   DDEV_COLOR: 1
+  DD_TRACE_AGENT_PORT: 8127
+
+resources:
+  containers:
+    - container: dd_agent
+      image: gcr.io/datadoghq/agent:latest
+      ports:
+        - 8127:8126
```

Comment:
Just curious, why is this `8127:8126` and not just `8127:8127`?

Reply:
The default agent port is `8126`, but this port is also used by some integration environment when starting the service ; so there is a port conflict.
`8127` is not used in our test, hence the `8127:8126`